Auto_Restart_Launcher.bat

```diff
@@ -0,0 +1,116 @@
+:://   7200 - 2 hours                                                     //
+:://   3600 - 1 hour                                                      //
+:://   1800 - 30 mins                                                     //
+:://   1200 - 20 mins   (recommended)                                     //
+:://   900 - 15 mins                                                      //
+:://   720 - 12 mins                                                      //
+:://   660 - 11 mins    (Don't recommend!)                                //
+:://////////////////////////////////////////////////////////////////////////
+
+set check_rate=720
+
+:://////////////////////////////////////////////////////////////////////////
+:://                  Server log location and filename                    //
+:://          Only change if you hosting more then 1 server               //
+:://      Unless you change g_log on server.cfg then match with that.     //
+:://////////////////////////////////////////////////////////////////////////
+
+set log_path=%cd%\Boiii
+set log_file=games_zm.log
+
+:://////////////////////////////////////////////////////////////////////////
+::// Below edits are optional unless you run multiable servers or mods.   //
+:://////////////////////////////////////////////////////////////////////////
+:://                 Load a mod on your server (Optional)                 //
+:://                   Example: ModfolderName=mods/bots                   //
+:://                                                                      //
+:://                   UNLOAD a mod on your server                        //
+:://                   Example: ModfolderName=                            //
+:://////////////////////////////////////////////////////////////////////////
+
+set ModFolderName=
+
+:://////////////////////////////////////////////////////////////////////////
+::// 	Your edited server.cfg in the "zone" folder goes here...          //
+:://	This is were you edit your hostname, rcon, inactivity, etc        //
+:://        (Only change if you hosting more then 1 server)               //
+:://////////////////////////////////////////////////////////////////////////
+
+set ServerFilename=server_zm.cfg
+
+:://////////////////////////////////////////////////////////////////////////
+:://  DONE!! WARNING! Don't mess with anything below this line. SEROUSLY! //
+:://             Only edit if you know what you doing!!!                  //
+:://////////////////////////////////////////////////////////////////////////
+
+// Create a copy of boiii.exe and rename it to something like boiiitdm.exe then edit the .exe names below to suit!
+title %name% - Auto Restart
+echo (%date%)  -  (%time%) Starting %name% with port %GamePort% with %ServerFilename%....
+start boiiitdm.exe -dedicated +set fs_game "%ModFolderName%" +set net_port "%GamePort%" +set logfile 2 +exec %ServerFilename%   //this line
```

Comment:
what's boiiitdm.exe? shouldn't it be boiii.exe?

Reply:
Reason for the renamed exe per server, Is due to the current crashing situation that is on going, if one server goes down with the same .exe name it tanked the rest of my servers